terraria-tmodloader/terraria-tmodloader.json

```diff
@@ -0,0 +1,222 @@
+{
+  "name": "terraria-tModLoader",
+  "display": "Terraria - tModLoader",
+  "type": "terraria",
+  "install": [
+    {
+      "files": [
+        "https://github.com/tModLoader/tModLoader/releases/download/v${tversion}/tModLoader.Linux.v${tversion}.tar.gz"
+      ],
+      "type": "download"
+    },
+    {
+      "commands": [
+        "mv ${rootDir}/${mp}/enabled.json ${rootDir}",
```

Comment:
RootDir is pretty much deprecated, as it's usage is very limited. Your commands should not need to use it, even though it works with it. Can you please remove them?

Reply:
I can remove them from the installation and the starting command, but when I remove them or replace them with ./ inside the configuration file, the server do not load the settings of the world's safe folder and file

Reply:
I found why it is not working without ${rootDir}, it is now removed

Reply:
I fixed it in all three templates